src/oom/oomd-manager.c

```diff
@@ -648,6 +650,26 @@ Manager* manager_free(Manager *m) {
         return mfree(m);
 }
 
+static int manager_dispatch_reload_signal(sd_event_source *s, const struct signalfd_siginfo *si, void *userdata) {
+        Manager *m = ASSERT_PTR(userdata);
+        int r;
+
+        (void) notify_reloading();
+
+        r = manager_set_defaults(m);
+        if (r < 0)
+                log_warning_errno(r, "Failed to set default values on reload: %m");
```

Comment:
If the error is ignored, then mention that in the log message:
`"Failed to set default values on reload, ignoring: %m"`